examples/nextjs-yjs-codemirror/package.json

```diff
@@ -11,9 +11,9 @@
   },
   "dependencies": {
     "@codemirror/lang-javascript": "^6.1.9",
-    "@liveblocks/client": "1.1.1-yjs3",
-    "@liveblocks/react": "1.1.1-yjs3",
-    "@liveblocks/yjs": "1.1.1-yjs3",
+    "@liveblocks/client": "^1.1.3",
+    "@liveblocks/react": "^1.1.3",
+    "@liveblocks/yjs": "^1.1.3",
```

Comment:
@jrowny Quick double-check with you. This is okay, right? All the stuff from `1.1.1-yjs3` is included in `1.1.2` too, I assume?

Reply:
yup, it should be fine